.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,61 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Run tests from tests/
+
+on:
+  push:
+    branches: [ "master", "*" ]
```

Comment:
Maybe these 2 lines should be commented out. This way, manually kicking off the workflow is possible whenever desired. And if I later change my mind and want it all the time automatically, I can just remove the 2 comment characters and be set. What do you think?

Reply:
I believe you can kick off the workflow manually already with these lines in.

The advantage of automating test runs on PRs is that you will be able to see if the tests passed when reviewing the MR.

For the main branch, it can make sense to run tests again when the proposed changes are merged (proving the resulting code), and it might make sense to run on a schedule (eg for a weekly integration test to validate connectivity with the latest matrix release).